openpype/hosts/blender/api/plugin.py

```diff
@@ -628,6 +629,7 @@ def _remove_instance(self, instance_name: str) -> bool:
 
         return True
 
+
 class Loader(LoaderPlugin):
```

Comment:
This class is useless, I think `AssetLoader` (renamed `BlenderLoader`) could be the one inheriting from `LoaderPlugin`